src/lib/modules/Loan/backoffice/LoanList/LoanListEntry.js

```diff
@@ -12,18 +12,15 @@ import { LoanDates } from './LoanDates';
 
```

Comment:
Sorry for being a naming police!
the parent folder should be named `LoanListEntry` not `LoanList` as the component name